File: pgm/ABC_Java/ABC_280_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        reader.readLine();
        final char sharp = '#';
        String line = "";
        int cnt = 0;
        while ((line = reader.readLine()) != null) {
            for (char c : line.toCharArray()) {
                if (c == sharp) {
                    cnt++;
                }
            }
        }
        System.out.println (cnt);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_280_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val sharp: Char = '#'

fun main() {
    readln()
    println(generateSequence { readlnOrNull() }.toList().sumOf { s: String ->
        s.count { c: Char -> c == sharp }
    })
    kotlin.system.exitProcess(0)
}